src/index.css

```diff
@@ -17,76 +17,6 @@ root:hover {
   color: var(--hover-blue);
 }
 
-.nav-container {
-  display: -webkit-box;
-  display: -ms-flexbox;
-  display: flex;
-}
-
-.nav-container nav {
-  background-color: var(--blue);
-  display: -webkit-box;
-  display: -ms-flexbox;
-  display: flex;
-}
-
-.nav-container nav .layout {
-  width: 70%;
-}
-
-.nav-container nav .layout [role='menu'] {
-  background-color: var(--blue);
-  height: 60px;
-  display: -webkit-box;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-box-pack: end;
-  -ms-flex-pack: end;
-  justify-content: flex-end;
-  -webkit-box-align: center;
-  -ms-flex-align: center;
-  align-items: center;
-}
-
-.nav-container .logo-container {
-  width: 11%;
-  height: 59px;
-  background-color: var(--blue);
-}
-
-.nav-container .lambda-container {
-  display: -webkit-box;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-box-align: center;
-  -ms-flex-align: center;
-  align-items: center;
-}
-
-.nav-container .lambda-container .lambda {
-  margin-top: 5px;
-  margin-left: 0.8rem;
-  padding-right: 1rem;
-  max-height: 2.5rem;
-}
-
-.nav-container .logo {
-  max-height: 3rem;
-  position: absolute;
-  z-index: +1;
-  margin-top: 0.25%;
-  margin-left: 0.25%;
-}
-
-nav {
-  display: inline;
-  width: 100%;
-}
-
-.active-nav-link {
-  font-weight: 600;
-}
-
```

Comment:
Nice catch here, consolidating all this Navbar styling into the NavBar.Less file where it belongs!